draft-ietf-teas-actn-poi-applicability.txt

```diff
@@ -1014,6 +1014,30 @@ Internet-Draft                 ACTN POI                   November 2020
    [Editor's note:] Add YANG models used for tunnel and service
    configuration.
 
+3.3. PCEP
+
+   [RFC8623] examines the applicability of a Path Computation Element
```

Comment:
RFC 8623 is       Stateful Path Computation Element (PCE) Protocol Extensions  for Usage with Point-to-Multipoint TE Label Switched Paths (LSPs)
Maybe you meant RFC8637

Reply:
**2020-11-09 TEAS**

Correct: it should be RFC8637